Bottom: 75bf3fb4b5e62e9215b9a8da602c17589176e84b
Top:    4f75bd010848d3dc93769f5f12ce94fd02f23697
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 18:51:38 +0000

polygon1.c: Rename rule and v_rule to j_rule for consistency


---

diff --git a/src/polygon1.c b/src/polygon1.c
index 5abef07..7f8e683 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -1540,7 +1540,7 @@ SubJ_Rule (char p, VNODE * v, DIRECTION * cdir)
  */
 /* *cur is considered a vertex */
 static int
-jump (VNODE ** cur, DIRECTION * cdir, J_Rule rule)
+jump (VNODE ** cur, DIRECTION * cdir, J_Rule j_rule)
 {
   CVCList *d, *start;
   VNODE *e; /* e is considered an edge */
@@ -1570,7 +1570,7 @@ jump (VNODE ** cur, DIRECTION * cdir, J_Rule rule)
       else
         e = VERTEX_FORWARD_EDGE (d->parent);
       newone = *cdir;
-      if (!e->Flags.mark && rule (d->poly, e, &newone))
+      if (!e->Flags.mark && j_rule (d->poly, e, &newone))
 	{
 	  if ((d->side == 'N' && newone == FORW) ||
 	      (d->side == 'P' && newone == BACKW))
@@ -1595,7 +1595,7 @@ jump (VNODE ** cur, DIRECTION * cdir, J_Rule rule)
 
 /* start is considered a vertex */
 static int
-Gather (VNODE * start, PLINE ** result, J_Rule v_rule, DIRECTION initdir)
+Gather (VNODE * start, PLINE ** result, J_Rule j_rule, DIRECTION initdir)
 {
   VNODE *cur = start, *newn; /* cur is considered a vertex */
   DIRECTION dir = initdir;
@@ -1606,7 +1606,7 @@ Gather (VNODE * start, PLINE ** result, J_Rule v_rule, DIRECTION initdir)
   do
     {
       /* see where to go next */
-      if (!jump (&cur, &dir, v_rule))
+      if (!jump (&cur, &dir, j_rule))
 	break;
       /* add edge to polygon */
       if ((newn = poly_CreateNode (cur->point)) == NULL)
